azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/CloningInfo.java

```diff
@@ -55,6 +55,12 @@
     @JsonProperty(value = "sourceWebAppId", required = true)
     private String sourceWebAppId;
 
+    /**
+     * Location of source app ex: West US or North Europe.
```

Comment:
Does this value mapping to the Region? If so, use region instead to avoid wrong typing of location

Reply:
This class is auto-generated. And we do not have wrapper class for it.